reduction/line_imaging_update.py

```diff
@@ -0,0 +1,251 @@
+if not os.path.exists(imaging_root):
+    os.mkdir(imaging_root)
+
+if 'exclude_7m' not in locals():
+    if os.getenv('EXCLUDE_7M') is not None:
+        exclude_7m = bool(os.getenv('EXCLUDE_7M').lower() == 'true')
+    else:
+        exclude_7m = False
+
+if os.getenv('LINE_NAME'):
+    line_name = os.getenv('LINE_NAME').lower()
+else:
+    raise ValueError("line_name was not defined")
+
+# set the 'chanchunks' parameter globally.
+# CASAguides recommend chanchunks=-1, but this resulted in:
+# 2018-09-05 23:16:34     SEVERE  tclean::task_tclean::   Exception from task_tclean :
+#Invalid Gridding/FTM Parameter set : Must have at least 1 chanchunk
+chanchunks = os.getenv('CHANCHUNKS') or 16
+
+# global default: only do robust 0 for lines
+robust = 0
+
+for band in band_list:
+    for field in to_image[band]:
+        for spw in to_image[band][field]:
+
+            vis = list(map(str, to_image[band][field][spw]))
+
+            if exclude_7m:
+                vis = [ms for ms in vis if not is_7m(ms)]
+                arrayname = '12M'
+            else:
+                arrayname = '7M12M'
+
+            lineimagename = os.path.join(imaging_root,
+                                         "{0}_{1}_spw{2}_{3}_{4}".format(field,
+                                                                         band,
+                                                                         spw,
+                                                                         arrayname,
+                                                                         line_name))
+
+
+            logprint(str(vis), origin='almaimf_line_imaging')
+            coosys, racen, deccen = determine_phasecenter(ms=vis, field=field)
+            phasecenter = "{0} {1}deg {2}deg".format(coosys, racen, deccen)
+            (dra, ddec, pixscale) = list(determine_imsizes(mses=vis, field=field,
+                                                         phasecenter=(racen, deccen),
+                                                         spw=0, pixfraction_of_fwhm=1/3.,
+                                                         exclude_7m=exclude_7m,
+                                                         min_pixscale=0.1, # arcsec
+                                                        ))
+            imsize = [int(dra), int(ddec)]
+            cellsize = ['{0:0.2f}arcsec'.format(pixscale)] * 2
+
+            dirty_tclean_made_residual = False
+
+
+            # prepare for the imaging parameters
+            pars_key = "{0}_{1}_{2}_robust{3}".format(field, band, arrayname, robust)
+            impars = line_imaging_parameters[pars_key]
+            # load in the line parameter info
+            linpars = line_parameters[line_name]
+
+            # calculate the channel width
+            count_spws = len(auIMF.spwsforfield(field+'_all_split.ms.contsub', field=field))
+            width = np.max([np.abs(auIMF.effectiveResolutionAtFreq(field+'_all_split.ms.contsub',
```

Comment:
Since this is the only tool we're using from analysisUtils, I prefer to take this (very small) function and put it into `metadata_tools`.  This should be straightforward - we'll just cut & past `effectiveResolutionAtFreq` into `metadata_tools.py`, and then figure out what dependencies are needed